src/Raven.Client/Documents/Smuggler/SmugglerResult.cs

```diff
@@ -113,6 +114,7 @@ public SmugglerProgress(SmugglerResult result)
                 Indexes = _result?.Indexes;
                 CompareExchange = _result?.CompareExchange;
                 Counters = _result?.Counters;
+                CompareExchangeTombstones = _result?.CompareExchangeTombstones;
```

Comment:
Need to do EnsureStepProcessed on this (check where the method is used)

Reply:
done